utils/walk.test.js

```diff
@@ -0,0 +1,68 @@
+const assert = require('assert');
+
+const bcd = require('./bcd');
+const { walk } = require('./index');
+const { lowLevelWalk } = require('./walk');
+
+describe('lowLevelWalk()', function () {
+  it('visits every top-level tree', function () {
+    const expectedPaths = [
+      'api',
+      'browsers',
+      'css',
+      'html',
+      'http',
+      'javascript',
+      'mathml',
+      'svg',
+      'webdriver',
+      'webextensions',
+      'xpath',
+      'xslt',
+    ];
+
+    const steps = Array.from(lowLevelWalk(undefined, undefined, 1));
+    const paths = steps.map(step => step.path);
+    assert.strictEqual(steps.length, expectedPaths.length);
+    assert.deepStrictEqual(paths, expectedPaths);
+  });
+  it('visits every point in the tree', function () {
+    const paths = Array.from(lowLevelWalk()).map(step => step.path);
+    assert.ok(paths.length > 13000);
```

Comment:
Strictly speaking, this doesn't _really_ test this function—just that its output is reasonable—but I feel like the tests on `walk()` do the heavy lifting here.